jetty-client/src/main/java/org/eclipse/jetty/client/util/AsyncRequestContent.java

```diff
@@ -291,15 +293,15 @@ private void notifyFailure(Consumer consumer, Throwable failure)
 
     private void notifyFlush()
     {
-        try (AutoLock.WithCondition l = lock.lock())
+        try (AutoLock l = lock.lock())
         {
-            l.signal();
+            flush.signal();
```

Comment:
The old code (on the left) called the signal on the AutoLock, this change does it on the flush.